src/Views/gagepage.html

```diff
@@ -240,20 +240,21 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.heatChartConfig">
                             <highchart id="chart2" config="vm.heatChartConfig"></highchart>
                         </div>
-                    </center> -->
+                    </center>
                     <center>
-                        <div ng-if="!vm.dischargeChartConfig" class="panel panel-default reportSpinner">
+                        <div ng-if="!vm.chartConfig" class="panel panel-default reportSpinner">
                             &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                         </div>
-                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.dischargeChartConfig">
-                        <input type="checkbox" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()">
-                        <label for="logScaleDischarge" > Log Scale</label>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
+
+                        <input type="checkbox" ng-if="vm.formattedDischargePeakDates.length > 0" id="logScaleDischarge" name="logScaleDischarge" ng-model="vm.logScaleDischarge" ng-change="vm.toggleLogLinearDischarge()" ng-hide="!data">
+                        <label for="logScaleDischarge" ng-if="vm.formattedDischargePeakDates.length > 0"> Log Scale</label>
                         
-                        <input type="radio" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')"> 
-                        <label for="ageData"> Age</label>
+                        <input type="radio"  ng-if="vm.measuredObj.control !== undefined" id="ageData" name="ageQualityData" ng-model="vm.ageQualityData" value="age" ng-change="vm.toggleDischargeData('age')" checked> 
```

Comment:
One of the Age or Quality buttons should be selected when you first open the graph. They aren't selected now because `this.ageQualityData` was deleted from the controller.

![image](https://user-images.githubusercontent.com/40237491/228267678-b47849d2-1676-411d-9b10-7550d664e371.png)